pkg/kubelet/secret/secret_manager.go

```diff
@@ -209,8 +209,12 @@ type cachingSecretManager struct {
 }
 
 func NewCachingSecretManager(kubeClient clientset.Interface) (Manager, error) {
+	// TODO: Currently apiserver completely cannot keep up with 1 minute ttl in
+	// 5000-node clusters. So before we optimize apiserver and/or etcd, increase
+	// the TTL to 2 minutes (which is a lot, but...)
+	ttl := 2 * time.Minute
```

Comment:
@yujuhong @liggitt - we have some result after merging the previous optimization. It helped, but the change isn't huge (i.e. everything now starts blowing at the level of ~128.000 pods, which is roughly 15.000 pods later).

That said, i think increasing a TTL would really help to solve this problem. I agree with all your previous arguments that you probably can't agree for 5m TTL.
However, based on some of my experiments and calculations, I think that increasing it by 2x should be enough for now.

So I modified the PR to bump the ttl to 2 minutes (instead of 5 minutes).

Is it something that you can accept? Can we proceed with this PR?